### ISSUE TITLE
Tele5 has an extraction error

### ISSUE THREAD

**Issue by kanehekili** [yt-dlp/yt-dlp#3183]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2022.03.08.1**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Germany

### Description

trying to download the curernt andromenda series:
`yt-dlp -F https://tele5.de/mediathek/gene-roddenberrys-andromeda/`
`[Tele5] gene-roddenberrys-andromeda: Downloading webpage`
`ERROR: gene-roddenberrys-andromeda: An extractor error has occurred. (caused by KeyError('assetid')); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest ver`

### Verbose log

```shell
ERROR: gene-roddenberrys-andromeda: An extractor error has occurred. (caused by KeyError('assetid')); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the "Broken site" issue template properly. Confirm you are on the latest version using  yt-dlp -U
  File "/usr/bin/yt-dlp/yt_dlp/extractor/common.py", line 617, in extract
    ie_result = self._real_extract(url)
  File "/usr/bin/yt-dlp/yt_dlp/extractor/tele5.py", line 81, in _real_extract
    asset_id, country, realm = (player_info[x] for x in ('assetid', 'locale', 'realm', ))
  File "/usr/bin/yt-dlp/yt_dlp/extractor/tele5.py", line 81, in <genexpr>
    asset_id, country, realm = (player_info[x] for x in ('assetid', 'locale', 'realm', ))
KeyError: 'assetid'
```

**Comment by pukkandan** [comment]
that is not verbose log!
```
❯ yt-dlp -F https://tele5.de/mediathek/gene-roddenberrys-andromeda/ -v
[debug] Command-line config: ['-F', 'https://tele5.de/mediathek/gene-roddenberrys-andromeda/', '-v']
[debug] Encodings: locale cp65001, fs utf-8, out utf-8, err utf-8, pref cp65001
[debug] yt-dlp version 2022.03.08.1 [c0c2c57d3] (source)
[debug] Lazy loading extractors is disabled
[debug] Plugins: ['SamplePluginIE', 'SamplePluginPP']
[debug] Git HEAD: ae7296264
[debug] Python version 3.10.3 (CPython 64bit) - Windows-10-10.0.22000-SP0
[debug] exe versions: ffmpeg N-105465-gbedb4bac1c-20220206 (setts,fdk), ffprobe N-105465-gbedb4bac1c-20220206, phantomjs 2.1.1
[debug] Optional libraries: brotli, certifi, Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] Using fake IP 53.252.185.66 (DE) as X-Forwarded-For
[debug] [Tele5] Extracting URL: https://tele5.de/mediathek/gene-roddenberrys-andromeda/
[Tele5] gene-roddenberrys-andromeda: Downloading webpage
ERROR: gene-roddenberrys-andromeda: An extractor error has occurred. (caused by KeyError('assetid')); please report this issue on  https://github.com/yt-dlp/yt-dlp , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
  File "D:\Programs\Source\yt-dlp\yt-dlp\yt_dlp\extractor\common.py", line 638, in extract
    ie_result = self._real_extract(url)
  File "D:\Programs\Source\yt-dlp\yt-dlp\yt_dlp\extractor\tele5.py", line 81, in _real_extract
    asset_id, country, realm = (player_info[x] for x in ('assetid', 'locale', 'realm', ))
  File "D:\Programs\Source\yt-dlp\yt-dlp\yt_dlp\extractor\tele5.py", line 81, in <genexpr>
    asset_id, country, realm = (player_info[x] for x in ('assetid', 'locale', 'realm', ))
KeyError: 'assetid'
```

**Comment by dirkf** [comment]
Same code as yt-dl.

We're looking for the `asset_id` attribute of this element, but it's no longer being sent:
```html
<hyoga-player
    uid="61ae314db0ed8d9799910ece"
    playerselector="hyogaPlayer-61ae314db0ed8d9799910ece"
    videolibrary="videojs"
    sourcetype="sonic"
    locale="de"
    playertype="showPlayer"
    showid="39808"
    seasonsorting="descending"
    episodesorting="descending"
    playbacktype="video"
    autoplay="false"
    muted="false"
    nextepisode="true"
    disableobserver="false"
    id="hyogaManager-61ae314db0ed8d9799910ece"
    adsystem="fw"
    endpoint="https://eu1-prod.disco-api.com"
    realm="dmaxde"
>
```
It's not `uid` or `showid` either, apparently, but the page doesn't load a video for me, perhaps geo or browser issue.

**Comment by kanehekili** [comment]
Well, they introduced DRM - so I guess you can  remove Tele5 from your list of supported sites

**Comment by dirkf** [comment]
Is that true for all shows, or just this and some others? The extractor expects shows to have DRM and reports it, but not by failing to have an `asset_id`.

**Comment by pukkandan** [comment]
Can someone confirm whether the posted URL has DRM? If yes, is there any non-DRM example of this issue? Not much point in trying to fix it otherwise (other than getting a nicer error message)

**Comment by kanehekili** [comment]
All links I've tried ended with KeyError: 'assetid'. So one must assume they switched to DRM. In german forums the date 2022-01-11 was mentioned.

**Comment by m4tu4g** [comment]
> Can someone confirm whether the posted URL has DRM? If yes, is there any non-DRM example of this issue? Not much point in trying to fix it otherwise (other than getting a nicer error message)

DRMed content

**Comment by pukkandan** [comment]
@joeyschulze and that gives non-DRM formats?

**Comment by joeyschulze** [comment]
This token allows you to access videoPlaybackInfo from disco-api.com which contains a link to a dash manifest and hls m3u8 which you may access using the same token as well.  This lets you watch the video.

**Comment by dirkf** [comment]
That looks like what `DPlayBaseIE._get_auth()` is doing, or `DPlayIE._update_disco_api_headers()` in yt-dl.

**Comment by Grub4K** [comment]
Implemented in 50e93e03a7ca6ae35a319ea310104f7d6d91eee3 with [`tele5.py`](https://github.com/yt-dlp/yt-dlp/commit/50e93e03a7ca6ae35a319ea310104f7d6d91eee3#diff-da311cd97bc5b5adb4cde1a3f29fe5bbb7e32ccfcfedeceee45f1d86e3f974ae).
